.github/workflows/conformance.yaml

```diff
@@ -83,3 +85,50 @@ jobs:
       - name: Debug failure
         if: failure()
         uses: ./.github/actions/kyverno-logs
+
+  # runs conformance test suites with configuration:
+  # - standard
+  # - force-failure-policy-ignore
+  force-failure-policy-ignore:
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-version:
+          - v1.24.7
+          - v1.25.3
+          - v1.26.0
```

Comment:
We should bump these up to the latest available patches of their respective minor versions.

Reply:
Yes, we will have to update branch protection though 😢